src/receiver.ts

```diff
@@ -55,12 +55,26 @@ export default class Receiver extends EventEmitter {
         email: string,
         password: string,
         idp: string,
-        clientCredentialsTokenStorageLocation?: string,
```

Comment:
The types did not complain about this change? I think I left it like this because the `generateCSSToken()` expects it